<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>asortiment</class>
 <widget class="QDialog" name="asortiment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>471</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="11">
    <widget class="QLabel" name="blabel_18">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="17">
    <widget class="QLabel" name="label7">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="blabel_3">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Назва</string>
     </property>
    </widget>
   </item>
   <item row="2" column="15">
    <widget class="QLabel" name="blabel_21">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="blabel">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Книга</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="blabel_4">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Автор</string>
     </property>
    </widget>
   </item>
   <item row="2" column="10">
    <widget class="QLabel" name="blabel_19">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label2">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="blabel_12">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QLabel" name="blabel_17">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="16">
    <widget class="QLabel" name="blabel_20">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="17">
    <widget class="QLabel" name="blabel_8">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ціна</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="blabel_9">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="14">
    <widget class="QLabel" name="blabel_7">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Кіл. стор.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="14">
    <widget class="QLabel" name="label6">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="blabel_10">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="12">
    <widget class="QLabel" name="label5">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLabel" name="blabel_14">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QLabel" name="label3">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="blabel_15">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="13">
    <widget class="QLabel" name="blabel_11">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="12">
    <widget class="QLabel" name="blabel_6">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Рік публікації</string>
     </property>
    </widget>
   </item>
   <item row="2" column="18">
    <widget class="QLabel" name="blabel_22">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QLabel" name="blabel_5">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Видавництво</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QLabel" name="blabel_16">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="9">
    <widget class="QLabel" name="label4">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="blabel_13">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="blabel_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="2" column="19">
    <widget class="QLabel" name="blabel_23">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Обкладинка</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="blabel_24">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="19">
    <widget class="QLabel" name="label8">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
